cypress/integration/settings/ChangePassword.feature

```diff
@@ -0,0 +1,31 @@
+Feature: Change password
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-03-23T01:05:01Z" title="Saturday, March 23rd 2019, 2:05:01 am +01:00">Mar 23, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,31 @@
+Feature: Change password
```


@datenbrei have a look into this user story + scenario